habitat-lab/habitat/datasets/rearrange/run_episode_generator.py

```diff
@@ -51,8 +51,7 @@ class RearrangeEpisodeGeneratorConfig:
         default_factory=lambda: ["data/objects/ycb/"]
     )
     # optionally correct unstable states by removing extra unstable objects (within minimum samples limitations)
-    # TODO: This option is off by default for backwards compatibility and because it does not yet work with target sampling.
-    correct_unstable_results: bool = False
+    correct_unstable_results: bool = True
```

Comment:
Is the backward compatibility comment not relevant anymore ?

Reply:
I don't feel this is relevant moving forward. We've been making other changes across the stack and previous versions have the old behavior if trying to reproduce (e.g. Habitat 2.0) exactly.